### ISSUE TITLE
asqwedd: The Tenants aren't the Only One

### ISSUE THREAD

**Issue by thegogcollector** [MachineGunnur/GOG-Games#387]
Hello,

*To MachineGunnur. It's a "DM" for asqwedd, not "truly" your issue, sorry. I forgot their RIN account name, don't have other means of contact, wrote to make 'em understand better. Please keep it for a while non-comment-closed for asqwedd, they aren't Watchers. Of course you can delete this message if you want, no biggie, seriously (I won't do anything about, this message should't be here to be clear). Mind you, I don't ask to fix it, I know you never will, no matter what I or others say. In total, the current site setup to me personally is sort of "manageable:" I'm in no hurry, able to get from other hosters or sites.*

asqwedd, The Tenants is a tiny part of the iceberg. There are many more games with that exact issue.

What you actually could want to fix is the root, not an accident of. The FAQ said a re-upload is available only once in two months, at best it might be an entire month until fixed despite the obvious fact that many hosters are only 1 month. The policy claims to protect against dead links, well, it actually makes dead links, a broken policy. I tried to loudly protest that political decision, gave complelling arguments, the way to cut traffic of "'false' re-uploads" when ZippyShare is not dead, well, no luck.

The truth to be told, unless all links are down, MachineGunnur won't listen. Maybe there is a really good reason why this or that such a way but I wasn't offered an insight, so I am not convinced. Please do scroll all the issues and read each survived, it can give an idea what a response to some things would be and if it's worth writing.

With regards,
The GOG Collector

**Comment by thegogcollector** [comment]
Nevermind, I found 'em. Sorry.

**Comment by asqwedd** [comment]
No idea what was said in this issue but it clearly mentioned my earlier report of dead links. If someone needs to find me, I'm still checking that old email address but very infrequently (there were weird non deliverable issues last time and some emails were bouncing back for no explainable reason). I'm also still on rin forums as asqwed.
